f8a_worker/workers/CVEchecker.py

```diff
@@ -222,12 +250,61 @@ def _clean_dep_check_tmp():
                 'status': 'success',
                 'details': results}
 
+    @staticmethod
+    def update_victims_cve_db_on_s3():
+        """Update Victims CVE DB on S3"""
+        repo_url = 'https://github.com/victims/victims-cve-db.git'
+        s3 = StoragePool.get_connected_storage('S3VulnDB')
+        with tempdir() as temp_data_dir:
+            victims_db_dir = os.path.join(temp_data_dir, 'victims-cve-db')
+            Git.clone(repo_url, victims_db_dir, depth="1")
+            s3.store_victims_db(temp_data_dir)
+
+    def _run_victims_cve_db_cli(self, arguments):
+        """Run Victims CVE DB CLI"""
+        s3 = StoragePool.get_connected_storage('S3VulnDB')
+        output = []
+
+        with tempdir() as temp_data_dir:
```

Comment:
you are using `tempdir` here, so `if not s3.retrieve_victims_db_if_exists(temp_data_dir)` will always be `True` since `tempdir` cleans after itself https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/utils.py#L155

Reply:
I don't understand this comment.

`s3.retrieve_victims_db_if_exists()` returns `True` if the to-be-retrieved file exists in S3 and `False` otherwise.

The `tempdir()` context manager cleans the directory *after* we leave the `with tempdir()` section.